catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/DeleteUserDefinedReceiverBrickTest.java

```diff
@@ -129,12 +128,7 @@ private void createProject(String projectName) throws IOException, CloneNotSuppo
 		createUserDefinedScripts();
 		createStartScript();
 
-		Scene scene = project.getDefaultScene();
-		project.getDefaultScene().addSprite(sprite);
-
-		copiedSprite = controller.copy(sprite, project, scene);
-
-		projectManager.setCurrentProject(project);
+		copiedSprite = controller.copy(sprite, project, project.getDefaultScene());
 		projectManager.setCurrentSprite(sprite);
```

Comment:
Please check if this function is executed in the TestUtils. If Yes please delete :pray:

Reply:
Is needed here again because an additional sprite is created in the test which is not part of the UiTestUtils.createDefaultTestProject function and therefore also needs to be added.